### DISCUSSION TITLE
CSRF mismatch error when trying to fetch login form info.

### DISCUSSION THREAD

**Discussion by MUzairS15** [ory/kratos#2810] [Q&A]
I am referencing this [example implementation ](https://github.com/atreya2011/go-kratos-test/tree/hydra-consent) to integrate hydra and Kratos.
The `login_challenge` and `flowid` is generated correctly but when I call `GetSelfServiceLoginFlow` to populate login form, the response has missing input fields, and I get 401 Unauthorised error.

Where as when I manually try to hit `self-service/login/flows?id=<flowid>` in this case flowId being generated by visiting `self-service/login/browser` (manually from the browser, i.e skipping login-challenge and hydra stuff)things work as desired, only when I integrate with hydra it doesn't work.

**Comment by vinckr** [comment]
Hello @MUzairS15 
The example implementation is outdated, I suggest to update to the latest version in Hydra and Kratos and try out the implementation how it is described here: https://github.com/ory/kratos/discussions/2976#discussioncomment-4507555
